agent/maestro_agent/services/jmeter/properties.py

```diff
@@ -81,6 +85,14 @@ def value_per_agent(value):
 
         return load_profile_prop
 
+    def get_run_properties(self):
+        run_properties = {
+            "maestro.api.host": MAESTRO_API_HOST + "/api/run_metrics",
+            "maestro.api.token": MAESTRO_API_TOKEN,
+            "maestro.run.id": self.run.id,
```

Comment:
would be good to also have a test when run_id is not None to make sure we always inject that value